corpora/csip/metadata/dmdsec/CSIP28/testCase.xml

```diff
@@ -21,6 +21,8 @@ The creation date and time of the referenced file. 1..1 MUST
     Rule needed for the existence of the attribute CREATED for element dmdSec/mdRef.
     ERROR if it does not exists.
   There is no need for a check for blank or NULL values in the attribute since the mets schema will only tolerate dateTime values</description>
+  <cardinality>1..1</cardinality>
```

Comment:
@PhillipAasvangTommerholt Sorry for being late to comment after the PR has already been merged, the attempted fix has somehow still landed at the wrong place in the XML, making it invalid to the test case schema at the moment. The elements are only valid inside `requirementText`, could you have another look? If you prefer I can also submit a follow-up PR.